scripts/config-loader.js

```diff
@@ -59,22 +57,37 @@ module.exports = function(source) {
     }
   });
 
+  // Resolve fields of type common_form_element.
+  config.place.place_detail.forEach(category => {
+    category.fields = category.fields.map(field => {
+      if (field.type === "common_form_element") {
+        return Object.assign(
+          {},
+          config.place.common_form_elements[field.name],
+          { name: field.name },
+        );
+      } else {
+        return field;
+      }
+    });
+  });
```

Comment:
I see this code is copy/pasted in the `static-build` script. Maybe we can consider defining this as a function and exporting that function so that we can import it into the `static-build` script? Or we can add the function into our `src/base/static/utils/config-utils.js` file.

Reply:
Yeah, there is definitely some jankiness here. I'll abstract this better before merging this PR in.